requirements.txt

```diff
@@ -1,5 +1,5 @@
-Django>=1.11,<1.11.999
-djangorestframework>=3.7,<3.7.999
-drf-nested-routers==0.90.0
+Django
+djangorestframework
+drf-nested-routers
```

Comment:
This change is unrelated to the overall change, and is probably unwanted. Limiting the range of allowed versions makes sure that future breaking changes don't break installations. Could you please just update the ranges to the updated ones?

Reply:
Ok, in this specific case I used the `requirements.txt` from the journal-mapper repository for the up to date dependencies, but I see your point.

Reply:
It's a slightly different usecase. There we want to encourage people to use the latest, so when they develop, they always test against latest. Here we are trying to create stability.